### PR TITLE
Add tests and extend documentation, some linting

### PR DESCRIPTION
**Pull request by nuest**
update docs

### CODE REVIEW

**Comment by nuest** [comment]
@JanKoppe ready to be merged!

File: README.md [hunk 1]
```diff
@@ -38,43 +36,65 @@ to the muncher container. If you do not want that, you can point muncher to a
...
 
-The o2r muncher only provides the main parts of the o2r web API. For example, serving the data (files) from the compendia is handled by o2r-contentbutler. To show a simple example implementation integrating both services, there is a additional compose file.
+The o2r muncher only provides the main parts of the o2r web API. For example, serving the data (files) from the compendia is handled by o2r-contentbutler. To show a simple example implementation integrating both services, there is a Docker compose configuration in the file `docker-compose/docker-compose.full.yml`.
 
-```
-cd docker-compose && docker-compose -f docker-compose.full.yml up
+```bash
+docker-compose -f docker-compose/docker-compose.full.yml up
 # after you're done, shutdown and delete all volumes (data):
 docker-compose -f docker-compose.full.yml down -v
```
Comment (JanKoppe) [inline README.md]: If you don't `cd` into the `docker-compose` folder, you wil also need to use `docker-compose -f docker-compose/docker-compose.full.yml up`.

File: README.md [hunk 2]
```diff
@@ -38,43 +36,65 @@ to the muncher container. If you do not want that, you can point muncher to a
...
 ```
+
+The API is then available at http://localhost/api/v1/compendium
+
+To inspect the database, run `docker network inspect dockercompose_default` (or find out the network name before with `docker network ls`) to find out the IP of the database container. Then connect to it (e.g. with adminMongo) using `mongodb://<ip>`.
+
 ## Testing
 
-Needs a completely new environment (empty database), preferably startet with the docker-compose files.
+Needs a completely new environment (empty database),which is preferably startet with the docker-compose files.
```
Comment (JanKoppe) [inline README.md]: *started

File: controllers/job.js [hunk 3]
```diff
@@ -81,7 +81,7 @@ exports.viewSingle = (req, res) => {
      * We also need additional features, like MIME type recognition, etc.
      */
   Job.findOne({id}).exec((err, job) => {
-    if (err || job == null) {
+    if (err || job === null) {
```
Comment (JanKoppe) [inline controllers/job.js]: This needs to be a loose comparison with `==`.

File: test/bagtainers/success-load/bag-info.txt [hunk 4]
```diff
@@ -1,3 +1,8 @@
 Bag-Software-Agent: bagit.py <http://github.com/libraryofcongress/bagit-python>
+<<<<<<< HEAD
```
Comment (JanKoppe) [inline test/bagtainers/success-load/bag-info.txt]: Git Merge noise that hasn't been cleaned up.

File: test/bagtainers/success-load/tagmanifest-md5.txt [hunk 5]
```diff
@@ -1,3 +1,9 @@
+<<<<<<< HEAD
```
Comment (JanKoppe) [inline test/bagtainers/success-load/tagmanifest-md5.txt]: Git Merge noise.

**Comment by nuest** [comment]
@JanKoppe Next try...

File: config/config.js [hunk 6]
```diff
@@ -34,12 +34,12 @@ c.mongo.collection = env.MUNCHER_MONGODB_COLLECTION || 'muncher';
...
 
 // fix mongo location if trailing slash was omitted
-if (c.mongo.location[c.mongo.location.length-1] !== '/') {
+if (c.mongo.location[c.mongo.location.length - 1] !== '/') {
   c.mongo.location += '/';
 }
 
 // fs paths
-c.fs.base       = env.MUNCHER_BASEPATH || '/tmp/muncher/';
+c.fs.base       = env.MUNCHER_BASEPATH || '/tmp/o2r-muncher/';
```
Comment (JanKoppe) [inline config/config.js]: If you change this path here, you also need to change it in the `docker-compose` files. Also, if we change this, shouldn't it be entirely project-specific, and not contain the name of _one_ implementation? I sugget `/tmp/o2r/`.

File: README.md [hunk 7]
```diff
@@ -38,46 +36,87 @@ to the muncher container. If you do not want that, you can point muncher to a
...
   Define on which Port muncher should listen. Defaults to `8080`.
-* `MUNCHER_MONGODB` __Required__
+- `MUNCHER_MONGODB` __Required__
   Location for the mongo db. Defaults to `mongodb://localhost/`. You will very likely need to change this.
-* `MUNCHER_MONGODB_COLLECTION`
+- `MUNCHER_MONGODB_COLLECTION`
   Which collection inside the mongo db should be used. Defaults to `muncher`.
-* `MUNCHER_BASEPATH`
+- `MUNCHER_BASEPATH`
   Base path for the compendia storage. Defaults to `/tmp/muncher`. If you want persistent compendia storage, you should point this to a separate volume.
```
Comment (JanKoppe) [inline README.md]: See comment for `config.js`.

**Pull request merged**